Anatoly_Katsubo/02.Git.Local/02.GIT.Local.md

```diff
@@ -0,0 +1,53 @@
+git init
```

Comment:
Could you please use git markdown for reports: [https://guides.github.com/features/mastering-markdown/](https://guides.github.com/features/mastering-markdown/)